autopilot/autopilot.py

```diff
@@ -46,7 +49,7 @@ def __connect_to_seeds(self):
                 # FIXME: better strategy than sleep(2) for building up
                 time.sleep(2)
         except:
-            pass
+            raise
```

Comment:
This comment seems to be on an outdated commit. This is no longer the behavior, the latest version specifically attempts to catch RPCError. I will squash the commits so that it is easier to view the change, and gets rid of unnecessary history.